packages/wrangler/src/__tests__/pages.test.ts

```diff
@@ -1823,12 +1823,6 @@ describe("pages", () => {
 			✨ Deployment complete! Take a peek over at https://abcxyz.foo.pages.dev/"
 			`);
 
-			expect(std.warn).toMatchInlineSnapshot(`
```

Comment:
Rather than deleting this check it might be good to just change the snapshot to show we render no warnings?

Reply:
✅